### PR TITLE
Laravel 10.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l10-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: The latest version of Laravel has a dependency for `phpunit/phpunit` of `^9.5.10`. If you would like to also upgrade your tests, you may run the [PHPUnit 9 Shift](https://laravelshift.com/upgrade-phpunit-9) for free.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-actions-ide-helper.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "wulfheart/laravel-actions-ide-helper": "dev-l10-compatibility",
    }
}
```

Finally, run: `composer update`

**Comment by edalzell** [comment]
FYI, this needs to be updated to pull in lody 0.4.0, so it'll run w/ the latest actions release

**Comment by edwinvdpol** [comment]
@Wulfheart Can you look into this by any chance?

**Comment by Wulfheart** [comment]
@edwinvdpol I hope I have time some time later this week. You can give it a go with the last comment from @laravel-shift and give me some notice if it works.

**Comment by vpratfr** [comment]
@Wulfheart I can confirm it works but it needs bumping lody to 0.4 (in addition to what shift has suggested)

**Comment by vpratfr** [comment]
https://github.com/laravel-shift/laravel-actions-ide-helper/compare/l10-compatibility...vpratfr:laravel-actions-ide-helper:l10-compatibility

File: composer.json [hunk 1]
```diff
@@ -16,25 +16,25 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "illuminate/contracts": "^9.0",
+        "php": "^8.1",
+        "illuminate/contracts": "^10.0",
         "lorisleiva/laravel-actions": "^2.3",
         "lorisleiva/lody": "^0.3.0",
```
Comment (vpratfr) [inline composer.json]:
```suggestion
        "lorisleiva/lody": "^0.4.0",
```

**Comment by Wulfheart** [comment]
Yolo

**Pull request merged**